stdlib/public/SwiftShims/CoreFoundationShims.h

```diff
@@ -69,6 +69,7 @@ _swift_shims_CFIndex _swift_stdlib_CFStringGetLength(
     _swift_shims_CFStringRef _Nonnull theString);
 
 SWIFT_RUNTIME_STDLIB_INTERFACE
+__attribute__((ns_returns_retained))
```

Comment:
It would be more efficient to annotate and remove the balancing release but I wanted to keep the change as small as possible for 4.0.